README.md

```diff
@@ -1,4 +1,53 @@
-# devtools-protocol
+# DevTools protocol
 
-This is the repository for devtools-protocol. You're welcome to
-[[contribute]](CONTRIBUTING.md)!
+This repository is for resources and discussion around aligning the developer tools wire protocol used by various browsers, JavaScript egines and debugging tools.
+
+## Goals
+
+- Help empower web developers to build great applications by supporting an ecosystem of diagnostics tools which work well across a variety of targets.
+- Reduce the need for target-specific "devtools protocol adapters" in order to simplify the experience for tool users and tool developers.
+- Provide a vendor-neutral forum to facilitate collaboration and consensus building between different protocol implementations and interested clients.
+
+[Contributions](CONTRIBUTING.md) from the community are welcome!
+
+## What is here in this repo
+
+Not much yet, just a [few issues](https://github.com/WICG/devtools-protocol/issues) tracking work we'd like to do and links to a few resources below.
+
+## Implementations
+Engines and browsers that all have a similar debugging protocol built-in:
+- [Chrome DevTools](https://chromedevtools.github.io/devtools-protocol/), also used by most other chromium-based browsers
+- [WebKit / Safari](https://github.com/WebKit/webkit/tree/master/Source/JavaScriptCore/inspector/protocol)
+- Firefox - [in development](https://groups.google.com/forum/#!msg/mozilla.dev.platform/4-4A8W-nP5g/Y9C9UkWTAAAJ)
+- [Node.js](https://chromedevtools.github.io/devtools-protocol/v8/)
```

Comment:
Let's add a line-item for Edge, e.g. (`Edge - [in-development; link forthcoming]`). I'll update it with a pointer to our EDP documentation once that's ready (which should be the next couple of weeks).

I'd like to ensure it's clear that Edge is actively working on this, as well.

Also, not sure if we want to refer here to the browsers which expose a DevTools Protocol stack, the engines which the DPs target, or the clients which facilitate working against those engines? I suppose said another way, should we differentiate more within "implementations" to separate browser, engine, and clients?

Reply:
Great, thanks.

I do think we want to differentiate between the browsers/engines and clients (since we expect just a few of the former, and that's likely where most of the value in consensus building lies).  I tried to separate those cases by having the list of implementations under the sub-heading of "engines and browsers ...", then "adapters", and a separate section for "clients".  I figured we probably didn't want to try to maintain a complete list of clients so just included a few links to other clients.  Feel free to suggest other changes to make it clearer, or just adjust this yourself after I land it - you're the expert here, not me ;-)